src/network/taraxa_capability.cpp

```diff
@@ -92,6 +92,12 @@ void TaraxaCapability::sealAndSend(NodeID const &nodeID, unsigned packet_type, R
     return;
   }
 
+  auto peer = getPeer(nodeID);
+
+  if (!peer->passed_initial_ && packet_type != StatusPacket) {
```

Comment:
I think this condition will break tests

Reply:
Maybe something like this https://github.com/Taraxa-project/taraxa-node/blob/develop/src/network/taraxa_capability.cpp#L213 could.  We should start to use something like `#ifdef TESTS` to distinguish between normal build and test build, but that's for the whole new task